src/tracer.js

```diff
@@ -128,7 +144,11 @@ export default class Tracer {
      * @return a REFERENCE_CHILD_OF reference pointing to `spanContext`
      */
     childOf(spanContext) {
```

Comment:
this is, strictly speaking, a static helper function, not a method of the tracer. Is it possible to export those separately, as `opentracing.childOf()`? Then from-scratch tracer implementations (like those written directly in ES5 to avoid babel dependencies) do not have to re-implement it.